aws/README.md

```diff
@@ -6,16 +6,30 @@
 3. [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
 
 ### Set Your Variables
-1. Clone this repository to your local machine.
-2. Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
+Uncomment and replace variables in `terraform.tfvars` with the desired values. Check `variables.tf` for default values.
 
 ### Deploy the Cluster
-From inside the cloned directory:
+From inside the `./aws` directory of this project:
 1. Run `terraform init`. This will initialize the terraform project.
 2. Run `terraform plan`. This will perform a dry run and find any problems with your configuration.
 3. Run `terraform apply`. This will deploy the cluster to AWS. Type 'yes' at the prompt.
 
-Congrats! you have a Kubernetes cluster! Next, [it's time to deploy Sourcegraph](https://docs.sourcegrapn.com/admin/deploy/kubernetes/configure)
+It will take a few minutes to deploy the cluster. When finished, you will be greeted with a success message.
+Congratulations! You have a Kubernetes cluster. Next, it's time to [deploy Sourcegraph](https://docs.sourcegraph.com/admin/deploy/kubernetes/configure).
+
+## Manage Your Cluster with Kubectl
+To manage your cluster, you can use the `kubectl` [command line tool](https://kubernetes.io/docs/tasks/tools/), which you'll need to configure cluster access for.
```

Comment:
I thought about it, but while I was testing these configs, I found that I really didn't configure kubectl until I'd already deployed the cluster. So I thought it would make more sense sequentially to place it here.

Reply:
Throughout the process of testing these configs, I found that this step was never taken before, but always after I provisioned the cluster. IMO, it is better-suited where it is.

Reply:
ah, fair